app/models/form_profile.rb

```diff
@@ -121,22 +125,11 @@ class FormProfile
   attribute :contact_information, FormContactInformation
   attribute :military_information, FormMilitaryInformation
 
-  # rubocop:disable Metrics/CyclomaticComplexity
   def self.prefill_enabled_forms
-    forms = []
-
-    forms += HCA_FORMS if Settings.hca.prefill
-    forms += PENSION_BURIAL_FORMS if Settings.pension_burial.prefill
-    forms += EDU_FORMS if Settings.edu.prefill
-    forms += VIC_FORMS if Settings.vic.prefill
-    forms << '21-686C'
-    forms << '40-10007'
-    forms += EVSS_FORMS if Settings.evss.prefill
-    forms += HLR_FORMS if Settings.decision_review.prefill
-
+    forms = %w[21-686C 40-10007]
+    ALL_FORMS.each { |type, form_list| forms += form_list if Settings[type].prefill }
```

Comment:
The reason I actually didnt do this very thing recently when I added in the HLR forms (and opted for the CyclomaticComplexity enable/disable) because I thought it was an odd way of accessing settings. It makes it less explicit in how prefill options are being added. But its a personal preference... I just like things to be explicit instead of clever sometimes :) but thats not the ruby way!